pkg/owner/id_test.go

```diff
@@ -30,3 +32,33 @@ func TestNewIDFromNeo3Wallet(t *testing.T) {
 	id := NewIDFromNeo3Wallet(wallet)
 	require.Equal(t, id.ToV2().GetValue(), wallet.Bytes())
 }
+
+func TestID_Parse(t *testing.T) {
+	t.Run("should parse successful", func(t *testing.T) {
+		for i := 0; i < 10; i++ {
+			t.Run(strconv.Itoa(i), func(t *testing.T) {
+				wallet, err := NEO3WalletFromPublicKey(&test.DecodeKey(1).PublicKey)
```

Comment:
For cycle I suggest to decode different keys with `test.DecodeKey(i).PublicKey`

Reply:
oh, you're right

Reply:
fixed